akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -180,12 +177,15 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
    * It can only be used for actors that were spawned by this `ActorTestKit`.
    * Other actors will not be stopped by this method.
    */
-  def stop[T](ref: ActorRef[T], max: FiniteDuration = timeout.duration): Unit = try {
-    Await.result(internalSystem ? { x: ActorRef[ActorTestKitGuardian.Ack.type] ⇒ ActorTestKitGuardian.StopActor(ref, x) }, max)
-  } catch {
-    case _: TimeoutException ⇒
-      assert(false, s"timeout ($max) during stop() waiting for actor [${ref.path}] to stop")
-  }
+  def stop[T](ref: ActorRef[T], max: FiniteDuration = timeout.duration): Unit =
+    try {
+      Await.result(internalSystem ? { x: ActorRef[ActorTestKitGuardian.Ack.type] =>
```

Comment:
Not sure that's how I would break lines here? If it's multiple lines, then rather each argument on an extra line.

Reply:
Depends on what follows it, e.g.
```scala
 def stop[T](ref: ActorRef[T], max: FiniteDuration = timeout.duration): Unit =
    try Await.result(internalSystem ? { x: ActorRef[ActorTestKitGuardian.Ack.type] =>
```
or

```scala
 def stop[T](ref: ActorRef[T], max: FiniteDuration = timeout.duration): Unit =
    try Await.result(internalSystem ? { 
      x: ActorRef[ActorTestKitGuardian.Ack.type] =>
```